blockchain/blockchain.go

```diff
@@ -272,6 +283,11 @@ func NewBlockChain(db database.DBManager, cacheConfig *CacheConfig, chainConfig
 			}
 		}
 	}
+
+	for i := 1; i <= runtime.NumCPU()/2; i++ {
```

Comment:
How about comparing severl goroutine number? 
```suggestion
	for i := 1; i <= runtime.NumCPU(); i++ {
```

Reply:
Let's merge this for now and then compare performance with different number of workers.

Reply:
Ok. Please test with different number.